pyproject.toml

```diff
@@ -15,6 +15,7 @@ requests = "^2.25.1"
 singer-sdk = "^0.9.0"
 intuit-oauth = "^1.2.4"
 target-hotglue = "^0.0.2"
+backoff = "^2.0.0"
```

Comment:
I could not install backoff locally
```
ERROR: Cannot install target-quickbooks and target-quickbooks==0.2.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    target-quickbooks 0.2.0 depends on backoff<3.0.0 and >=2.0.0
    singer-sdk 0.9.0 depends on backoff<2.0 and >=1.8.0
```

`singer-sdk` uses `backoff = ">=1.8.0,<2.0"`